Formula/ruby@2.3.rb

```diff
@@ -1,9 +1,9 @@
 class RubyAT23 < Formula
   desc "Powerful, clean, object-oriented scripting language"
   homepage "https://www.ruby-lang.org/"
-
```

Comment:
let's leave the line here :)

Reply:
You're going to give me nervous twitches you know :wink:

Reply:
That is indeed the goal.

Reply:
Please, I'm unbalanced enough as it is.